test/mailers/previews/user_mailer_preview.rb

```diff
@@ -113,6 +113,13 @@ def change_email
     UserMailer.change_email(user.id, old_email, new_email)
   end
 
+  def change_username
+    user = create(:user, :for_mailer_preview)
+    user.renamed_at = Time.current
```

Comment:
```suggestion
    user = create(:user, :for_mailer_preview, renamed_at: Time.current)
```

Reply:
I tried that, but it raises a validation error when I do that